Graphviz/Completions/Attributes.sublime-completions

```diff
@@ -1,177 +1,1209 @@
 {
-    "scope": "source.dot meta.attributes - string",
+    "scope": "source.dot & (meta.group | meta.attributes) - meta.string - string.quoted.double",
```

Comment:
Additionally `- meta.string` causes completions to be supressed, if GraphViz was embedded into a string of another language (`source.foo meta.stringl.foo meta.interpolation.foo source.dot`).